packages/database/data-migrations/20240612115151_adds_product_config/data-migration.ts

```diff
@@ -0,0 +1,100 @@
+import { PrismaClient, SurveyType } from "@prisma/client";
+
+const prisma = new PrismaClient();
+
+type channelType = Exclude<SurveyType, "web"> | null;
```

Comment:
Can we export the `TProductConfigChannel` type and use it here to be more consistent?
(even though it currently works; but unsure if it will stay like this in the future)